sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -276,8 +276,17 @@
 * TODO: This style has been added as v-dialog is not being responsive. 
 * Once This bug is resolved in Vuetify3, this style can be removed.
 */
-.restriction-modal {
+.temporary-modal {
   min-width: 600px;
+  max-height: 700px;
+  overflow: auto;
+}
+
+.locations-grid .table td,
+.locations-grid .table-bordered {
+  padding-top: 0 !important;
```

Comment:
for my understanding, why this style

Reply:
While using a form data grid, it is applying border for every row and padding it. In case of approve designation the border and the padding has to be removed to removed to make the locations list consume only sufficient space.
 like this. 
![image](https://user-images.githubusercontent.com/54600590/156661841-3cbd3d5b-16a1-4b75-a5ee-e11a1efaebb0.png)

Reply:
I will add comments to it for better understanding.

Reply:
okay, `locations-grid` is the custom class you added right?

Reply:
Yes @dheepak-aot.